man/systemd.unit.xml

```diff
@@ -115,6 +115,14 @@
     <para>Unit files are loaded from a set of paths determined during
     compilation, described in the next section.</para>
 
+    <para>Unit files can be parameterized by a single argument called the
+    "instance name". The unit is then constructed based on a "template file"
```

Comment:
else where we called this "instance string" rather than "instance name"... And I think that's better wording, since if you ask me "instance name" would be the full name, i.e. "foo@bar.service"...

Reply:
Actually we don't ;) But I'll do this, and also update other places.

Reply:
Hmm, no, I think "instance name" is a nicer name. Instance string just doesn't sound right. I'll leave it as "instance name" for now, and we can then discuss changing it everywhere.